common/djangoapps/entitlements/migrations/0010_backfill_entitlement_refundability.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from datetime import datetime
+
+from django.db import migrations, models
+
+
+def backfill_refundability(apps, schema_editor):
+    CourseEntitlementSupportDetail = apps.get_model('entitlements', 'CourseEntitlementSupportDetail')
+    for support_detail in CourseEntitlementSupportDetail.objects.all().select_related('entitlement'):
```

Comment:
Would this miss the case deen mentioned during backfill? does it matter? 
this would set marked_unrefundable on entitlements with support actions regardless of whether those entitlements were refundable before the support action.

Reply:
Yeah, we decided it would be fine for the sake of backfill to just mark them all as unrefundable